UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -48,16 +50,24 @@ public class EnergyTests : EnergyTestsBase
 
         protected override double GigawattHoursInOneJoule => 2.77777778e-13;
 
+        protected override double GigawattDaysInOneJoule => 1.157407407407410E-14;
+
         protected override double KilocaloriesInOneJoule => 0.00023900573614;
 
         protected override double KilowattHoursInOneJoule => 2.77777778e-7;
 
+        protected override double KilowattDaysInOneJoule => 1.157407407407410E-08;
+
         protected override double MegawattHoursInOneJoule => 2.77777778e-10;
 
+        protected override double MegawattDaysInOneJoule => 1.157407407407410E-11;
+
         protected override double ThermsUsInOneJoule => 9.4804342797334860315281322406817e-9;
 
         protected override double WattHoursInOneJoule => 0.000277777778;
 
+        protected override double WattDaysInOneJoule => 1.157407407407410E-05;
```

Comment:
I couldn't find a source on this, but WattHours value / 24 give this so looks good to me.